src/algorithms/tonal/chordsdetectionbeats.cpp

```diff
@@ -27,7 +27,12 @@ namespace standard {
 
 const char* ChordsDetectionBeats::name = "ChordsDetectionBeats";
 const char* ChordsDetectionBeats::category = "Tonal";
-const char* ChordsDetectionBeats::description = DOC("This algorithm estimates chords using pitch profile classes on segments between beats. It is similar to ChordsDetection algorithm, but the chords are estimated on audio segments between each pair of consecutive beats.\n"
+const char* ChordsDetectionBeats::description = DOC(
+"This algorithm estimates chords using pitch profile classes on segments between beats. "
+"It is similar to ChordsDetection algorithm, but the chords are estimated on audio segments between each pair "
+"of consecutive beats. Chroma vector from the segment could be obtained as:\n"
+"  - 'interbeat_median', each resulting chroma vector component is a median of all the component values in the segment\n"
+"  - 'first_beat', chroma vector is sampled from the segment's first beat (makes sense if chroma is preliminary smoothed)\n"
```

Comment:
All segment are covering audio between two consecutive beats. The "segment's starting beat" would be more clear. Maybe something like this? "Chroma vector is sampled from the start of each segment (that is, its starting beat position) using the first frame in each segment."

Reply:
Ok, It's better. I use this phrasing and rename first_beat to starting_beat (committed)